src/EditorFeatures/Core/Implementation/SplitComment/SplitCommentCommandHandler.cs

```diff
@@ -92,7 +92,7 @@ public bool ExecuteCommand(ReturnKeyCommandArgs args, CommandExecutionContext co
 
                 subjectBuffer.Replace(result.Value.replacementSpan, result.Value.replacementText);
 
-                transaction.Complete();
+                transaction?.Complete();
```

Comment:
Yay, actual bugs being found!

Reply:
Not sure this was a bug in practice, but #48676 was found and fixed by this